### ISSUE TITLE
Where are the application settings stored in macOS?

### ISSUE THREAD

**Issue by SmetDenis** [amnezia-vpn/amnezia-client#1244]
I updated using `AmneziaVPN_4.8.2.3.dmg`, Mac OS Sonoma 14.5 and lost all the settings. The app is blank. Sorry, I can't remember recent app version.

I have backups using Time Machine.

Can you tell me where the client setting are stored?
I want to restore the backup to get my settings back.

Thanks you so much!

**Comment by mchumakov** [comment]
Did you found those settings in macOS?

**Comment by SmetDenis** [comment]
Nope. Unfortunately I had to recreate the accesses and the servers. Now I make backups manually.

**Comment by variatto** [comment]
Found all configs at **/Users/%user%/Library/Preferences/**
Copied from timemachine backup:

- **AmneziaVPN.plist**
- **e.AmneziaVPN.plist**
- **org.amneziavpn.AmneziaVPN.plist**

And copied two keys from the KeyChain with name **AmneziaVPN-Keychain**, where stored keys for decode keys in Plists.

And all servers and preferences transfered and working.
